lib/paged_iteration.js

```diff
@@ -200,17 +191,15 @@ PageDescriptor.prototype.createStream = function(
     if ('pageToken' in options) {
       delete options.pageToken;
     }
-    return apiCall(response[1], options)
-        .then(callback)
-        .catch(function(err) {
-          stream.emit('error', err);
-        });
+    if (stream.isPaused()) {
+      request = next;
+    } else {
+      nextTick(apiCall, next, options, callback);
+    }
   }
-  apiCall(request, options)
-      .then(callback)
-      .catch(function(err) {
-        stream.emit('error', err);
-      });
+  stream.on('resume', function() {
+    apiCall(request, options, callback);
```

Comment:
You're right. Uploaded the fix for that with a new test case to capture the situation.